### PR TITLE
add blocker for import button

### PR DESCRIPTION
**Pull request by eddiebroad**
per DSDEEPB-1160 comment

### CODE REVIEW

**Comment by MattPutnam** [comment]
I think in this case it's okay to just pass state.  We're already passing props and refs.  This isn't a "shared state" thing like previously discussed, the `render-import-button` function is merely a convenience to keep the code clear.

**Comment by eddiebroad** [comment]
Re: "I think in this case..." I passed and used set! with ":importing?" via that.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs [hunk 1]
```diff
@@ -30,24 +30,27 @@
...
-                              "destinationNamespace" dest-conf-namespace
-                              "destinationName" dest-conf-name}
-                    :on-done (fn [{:keys [success? xhr]}]
-                               (if success?
-                                 (on-import {"name" dest-conf-name "namespace" dest-conf-namespace})
-                                 (js/alert (str "Import Error: " (.-responseText xhr)))))}))}]))
+      :onClick (fn []
+                 (let [dest-conf-name (-> (@refs "destinationName") .getDOMNode .-value)
+                       dest-conf-namespace (-> (@refs "destinationNamespace") .getDOMNode .-value)]
+                   ((fn [] (swap! state assoc :importing? true)))
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs]: Why are you creating a function and then immediately calling it?

Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs]: to achieve the desired function with the passed-in state.
Would it be preferred to use a  let binding here?

Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_importer.cljs]: No... just write `(swap! ...)`

**Comment by eddiebroad** [comment]
Re : "No... just write (swap! ...)"  Now, it's called more directly with less "wrapper" overhead.

**Pull request merged**